src/Mesh/Topologies.jl

```diff
@@ -1052,6 +1052,74 @@ function grid_stretching_1d(coord_min, coord_max, Ne, stretching_type, attractor
     return range_stretched
     
 end
-#}}}
 
+
+
+
+function Generate_grid(::Val{3},DDims,ts::Float64,tf::Int64,mpicomm::MPI.Comm,O::Int64,DomainSize,periodicity,Boundary)
+        #Problem type
+        DT = Float64
+        #DG Polynomial Order
+        polynomialorder=O
+        deltax=DDims[1]
+        deltay=DDims[2]
+        deltaz=DDims[3]
+        #Physical Domain Extents
+        (xmin,xmax)=DomainSize[1]
+        (ymin,ymax)=DomainSize[2]
+        (zmin,zmax)=DomainSize[3]
+        #Get Nex,Ney from resolution
+        Lx=xmax-xmin
+        Ly=ymax-ymin
+        Lz=zmax-zmin
+        #User defines the grid size
+        Nex = ceil(Int64, (Lx/deltax-1)/polynomialorder)
+        Ney = ceil(Int64, (Ly/deltay-1)/polynomialorder)
+        Nez = ceil(Int64, (Lz/deltaz-1)/polynomialorder)
+        Ne=(Nex,Ney,Nez)
+	# User defined domain parameters
+        brickrange = (range(DT(xmin), length=Ne[1]+1, DT(xmax)),
+                      range(DT(ymin), length=Ne[2]+1, DT(ymax)),
+                      range(DT(zmin), length=Ne[3]+1, DT(zmax)))
+        period=periodicity
+        bound=Boundary
+        
+        topl = StackedBrickTopology(mpicomm, brickrange,periodicity = period, boundary=bound)
+        dt = ts
+        timeend = tf
```

Comment:
time related variables (e.g. `dt` and `timeend`) probably don't need to be in here, since they aren't used.